### ISSUE TITLE
This doesn't work with esp32

### ISSUE THREAD

**Issue by tusharvb19** [esp8266/ESPWebServer#1]
fatal error: ESP8266WiFi.h: No such file or directory!

**Comment by bbx10** [comment]
This project roadmap may be helpful.

https://github.com/espressif/arduino-esp32/projects/1

**Comment by tablatronix** [comment]
Not really, it carded in development, but there is no activity whatsoever here. Is this abandoned?
I know about bbx10 fork but , no PR or anyhing
